Update api structure (#671)
nervosnetwork/docs.nervos.org: website/docs/script/js/02-04-js-api-other-utilities.mdx

# Other Utilities

## mount

Mount a file system

```ts
function mount(codeHash: ArrayBuffer, hashType: number): void;
```

**Parameters**

- `codeHash`: The code hash of the Cell to mount.
- `hashType`: The hash type of the code (use SCRIPT*HASH_TYPE*\* constants).

## evalJsScript

Evaluate a JavaScript Script (Generally used before `loadJsScript`).

```ts
function evalJsScript(jsScript: string, enableModule?: boolean): any;
```

**Parameters**

- `jsScript`: The JavaScript Script to evaluate
- `enableModule`: Whether to enable ES6 module (default: false)

**Return**

- The result of the Script evaluation. When `enableModule` is true, the Script doesn't return any value.

## loadJsScript

Load a JavaScript Script

```ts
function loadJsScript(path: string, enableModule?: boolean): any;
```

**Parameters**

- `path`: The path to the JavaScript Script
- `enableModule`: Whether to enable ES6 module (default: false)

**Return**

- The loaded Script as ArrayBuffer.
- When `enableModule` is true, the Script doesn't return any value.

## loadFile

Load a file

```ts
function loadFile(path: string): string;
```

**Parameters**

- `path`: The path to the file

**Return**

> - `string`: The loaded file as string

Reviewer: ```suggestion
- `string`: The loaded file as a string
```

## debug

> Output debug message.

Reviewer: ```suggestion
Output a debug message.
```

```ts
function debug(message: string): void;
```

**Parameters**

- `message`: The debug message to output

:::note

> Although calling `debug` directly will have better performance, it is still recommended to use `console`.

Reviewer: ```suggestion
While calling `debug` directly may offer better performance, using `console` is generally recommended unless you are in a highly performance-sensitive situation.
```
